### PR TITLE
Add Simple PBR Materials

### PR DESCRIPTION
**Pull request by sebavan**

### CODE REVIEW

File: src/babylon.scene.ts [hunk 1]
```diff
@@ -189,6 +189,25 @@
...
+         * Texture used in all pbr material as the reflection texture.
+         * As in the majority of the scene they are the same (excpetion for multi room and so on),
+         * this is easier to reference from here than from all the materials.
+         */
+        public get environmentTexture(): BaseTexture {
+            return this._environmentTexture;
+        }
+        /**
+         * Texture used in all pbr material as the reflection texture.
+         * As in the majority of the scene they are the same (excpetion for multi room and so on),
```
Comment (deltakosh) [inline src/babylon.scene.ts]: tiny typo

**Comment by sebavan** [comment]
Fixed #2209

**Comment by sebavan** [comment]
Add a first step towards #2009 with scene environment texture.

**Comment by sebavan** [comment]
Adding createDefaultSkybox fixes #2007

**Comment by benaadams** [comment]
Item for what's new on environment texture?

**Comment by deltakosh** [comment]
What do you mean? :)

**Comment by benaadams** [comment]
`scene.environmentTexture = ` is new, should it be in `what's new.md`?

**Comment by benaadams** [comment]
`scene.environmentTexture = ` is new, should it be in `what's new.md`?

Was looking to do something similar, then found this PR :)

**Comment by deltakosh** [comment]
TOTALLY! This is @sebavan  fault 🗡 
Thanks for mentioning it!

**Comment by deltakosh** [comment]
https://github.com/BabylonJS/Babylon.js/issues/2348

**Comment by sebavan** [comment]
Yup, my bad... I will do a big pass on doc and what s new before 3.0.

**Pull request merged**